proj/metals.conf

```diff
@@ -0,0 +1,8 @@
+// https://github.com/scalameta/metals.git#main
+
+vars.proj.metals: ${vars.base} {
+  name: "metals"
+  uri: "https://github.com/scalameta/metals.git#c2c6b7077f46b9d30e9be3806691e792919bfa92"
+
+  extra.exclude: ["sbt-metals", "input3", "bench", "docs"]
```

Comment:
I'm not familiar with community build but I if it's possible it would be better to include only `mtags` and `cross` subprojects of metals build. + run tests only for `cross`

Metals itself and it's other test-suprojects have some flaky tests + some tests data highly depends on scala-compiler version and might lead to false-positive failures

Reply:
Okay, thanks, I'll keep that in mind. I'll put some version of this in a comment, actually.